activestate.generators.yaml

```diff
@@ -56,9 +56,13 @@ scripts:
     description: Generate update files
     value: |
       set -e
+      export GOARCH=${1:-amd64}
       $constants.SET_ENV
 
-      export GOARCH=${1:-amd64}
+      # Generate legacy update
+      go build -o $BUILD_TARGET_DIR/$constants.BUILD_TRANSITION_TARGET ./cmd/state-transition-update
```

Comment:
I think we'll need to sign this on Windows, otherwise we might get the UAC prompt?

Reply:
Good point!  I think I'll cramp that in there too, as we are deleting that code soon anyways...

Reply:
Shoot. That won't work, because signing is only done on CI.  Okay, then I have to split it up.